chef/site-cookbooks/wca/recipes/cronjobs.rb

```diff
@@ -1,24 +1,29 @@
 username, repo_root = WcaHelper.get_username_and_repo_root(self)
+secrets = WcaHelper.get_secrets(self)
 
 admin_email = "admin@worldcubeassociation.org"
 path = "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games"
 
-db_dump_folder = "#{repo_root}/secrets/wca_db"
-dump_command = "#{repo_root}/scripts/db.sh dump #{db_dump_folder}"
+secrets_folder = "#{repo_root}/secrets"
+db_dump_folder = "#{secrets_folder}/wca_db"
+dump_db_command = "#{repo_root}/scripts/db.sh dump #{db_dump_folder}"
+dump_gh_command = "github-backup --incremental --all -t #{secrets['GITHUB_BACKUP_ACCESS_TOKEN']} --organization thewca -o #{secrets_folder}/github-thewca"
+backup_command = "#{dump_db_command} && #{dump_gh_command}"
```

Comment:
I would probably call these `backup_db_command` and `backup_gh_command` to avoid different terminology, but I don't see a problem with `dump.